src/ui/public/notify/notifier.js

```diff
@@ -63,7 +63,7 @@ function timerCanceler(notif, cb = _.noop, key) {
 function startNotifTimer(notif, cb) {
   const interval = 1000;
 
-  if (notif.lifetime === Infinity || notif.lifetime === 0) {
+  if (notif.lifetime === 'Infinity' || notif.lifetime === Infinity || notif.lifetime === 0) {
```

Comment:
I can appreciate that, but I think the benefit of doing it here is that every place that uses the advanced config options can ignore the output and hand it off here. Unless we want to provide a getter on the config object that can parse this for every caller.